api/HTMLMediaElement.json

```
@@ -2668,16 +2668,26 @@
             "edge": {
               "version_added": "86"
             },
-            "firefox": {
-              "prefix": "moz",
-              "version_added": "20",
-              "notes": "See <a href='https://bugzil.la/1652950'>bug 1652950</a>."
-            },
-            "firefox_android": {
-              "prefix": "moz",
-              "version_added": "20",
-              "notes": "See <a href='https://bugzil.la/1652950'>bug 1652950</a>."
-            },
+            "firefox": [
+              {
+                "version_added": "101"
+              },
+              {
+                "prefix": "moz",
+                "version_added": "20",
+                "notes": "Deprecated as of version 101. See <a href='https://bugzil.la/1652950'>bug 1652950</a> and <a href='https://bugzil.la/1765201'>bug 1765201</a>."
```

I think linking to just https://bugzil.la/1765201 here will suffice. I tried to figure out in which sense this is deprecated, and neither reading nor writing the attribute results in a console warning. To not risk confusion with BCD's notion of deprecated (which is per-feature and not per-browser) I think we can make this note just:

```suggestion
                "notes": "May be removed in a future release, see<a href='https://bugzil.la/1765201'>bug 1765201</a>."
```